### PR TITLE
Adding notify_time and time-reconnect options to client config

### PR DESCRIPTION
**Pull request by rafaelfc-olx**
While testing wazuh-agent behind an ELB in AWS, I experienced lots of disconnected agents after registration. After digging for a while, I realized we'd need to set notify_time and time-reconnect to values below the ELB idle timeout setting (default 60, can be adjusted up to 3600 seconds).

Hope this will helps https://github.com/wazuh/wazuh/issues/139 somehow.

This change is also backwards-compatible, as the default values for the settings are preserved.

### CODE REVIEW

**Comment by jnarvaezp** [comment]
Hi @rafaelfc-olx 
Thanks for your contribution, let me review and merge!

File: manifests/client.pp [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
+  $ossec_rootcheck             = true,
+  $ossec_rootcheck_frequency   = 36000,
+  $ossec_rootcheck_checkports  = true,
+  $ossec_rootcheck_checkfiles  = true,
+  $ossec_server_ip             = undef,
+  $ossec_server_hostname       = undef,
+  $ossec_server_port           = '1514',
+  $ossec_server_protocol       = 'udp',
+  $ossec_server_notify_time    = '600',
+  $ossec_server_time_reconnect = '1800',
```
Comment (jnarvaezp) [inline manifests/client.pp]: Hi @rafaelfc-olx 

These proposed values are of great use for users, but it is better to leave these "undef" variables for each user to make their modification and use the default values that are in the installation.

JN

Comment (rafaelfc-olx) [reply manifests/client.pp]: sure, makes sense. BTW these are the current defaults (https://documentation.wazuh.com/2.0/user-manual/reference/ossec-conf/client.html#time-reconnect and https://documentation.wazuh.com/2.0/user-manual/reference/ossec-conf/client.html#notify-time), but its safer to leave undef by default for future releases.

Thanks

**Comment by jnarvaezp** [comment]
Hi @rafaelfc-olx 

I've tested the changes, and they work properly.

Thanks for the changes and contribution.

JN

**Pull request merged**